core/src/test/java/com/sequenceiq/cloudbreak/logger/MDCBuilderTest.java

```diff
@@ -12,13 +12,15 @@
 
     @Test
     public void buildSimpleContext() {
+        MDC.clear();
```

Comment:
These changes seem unrelated to the ticket. Please write details into the commit message why it was needed. 
Also, it could be moved to a `@Before` method.

Reply:
The test failed on Linux without clearing the MDC context.

Reply:
Could you pls use `@Before` then, @bbihari ?

Reply:
I will remove this file from this pull and create a separate pull request